src/views/Completed.vue

```diff
@@ -34,65 +33,50 @@
-              Darooty Magwood
-              <p>{{ $t("Ordered") }} 27th January 2020 9:24 PM EST</p>
+              {{ order.doclist.docs[0]?.customerPartyName }}
+              <p v-if="order.doclist.docs[0]?.orderDate">
+                {{ $t("Ordered") }} {{ $filters.formatDate(order.doclist.docs[0]?.orderDate) }}
+              </p>
             </ion-label>
           </div>
 
-          <div class="order-tags">
+          <div class="order-tags" v-if="order.doclist.docs[0]?.orderId">
             <ion-chip outline>
               <ion-icon :icon="pricetagOutline" />
-              <ion-label>NN10584</ion-label>
+              <ion-label>{{ order.doclist.docs[0]?.orderId }}</ion-label>
             </ion-chip>
           </div>
 
           <div class="order-metadata">
             <ion-label>
-              Next Day Shipping
-              <p>{{ $t("Ordered") }} 28th January 2020 2:32 PM EST</p>
+              {{ order.doclist.docs[0].shipmentMethodTypeId }}
+              <!-- TODO: handle for this property -->
+              <p>{{ $t("Last brokered") }} 28th January 2020 2:32 PM EST</p>
             </ion-label>
           </div>
         </div>
 
-        <div class="order-item">
-          <div class="product-info">
-            <ion-item lines="none">
-              <ion-thumbnail>
-                <img src="https://dev-resources.hotwax.io/resources/uploads/images/product/m/j/mj08-blue_main.jpg" />
-              </ion-thumbnail>
-              <ion-label>
-                <p class="overline">WJ06-XL-PURPLE</p>
-                Juno Jacket
-                <p>Blue XL</p>
-              </ion-label>
-            </ion-item>
-          </div>
-
-          <div class="product-metadata mobile-only">
-            <ion-note>49 {{ $t("pieces in stock") }}</ion-note>
-          </div>
-        </div>
-
-        <div class="order-item">
+        <div class="order-item" v-for="(item, index) in order.doclist.docs" :key="index">
           <div class="product-info">
             <ion-item lines="none">
               <ion-thumbnail>
-                <img src="https://dev-resources.hotwax.io/resources/uploads/images/product/m/j/mj08-blue_main.jpg" />
+                <Image :src="getProduct(item.productId).mainImageUrl" />
               </ion-thumbnail>
               <ion-label>
-                <p class="overline">WJ06-XL-PURPLE</p>
-                Juno Jacket
-                <p>Blue XL</p>
+                <p class="overline">{{ item.internalName }}</p>
+                {{ getProduct(item.productId).parentProductName }}
+                <p> {{ $filters.getFeature(getProduct(item.productId).productFeatures, 'Color') }} {{ $filters.getFeature(getProduct(item.productId).productFeatures, 'Size') }} </p>
               </ion-label>
             </ion-item>
           </div>
 
           <div class="product-metadata mobile-only">
-            <ion-note>49 {{ $t("pieces in stock") }}</ion-note>
+            <!-- TODO: Handle for this property -->
```

Comment:
I think making stock quantity is handled, so there is no need for this TODO.

Reply:
Sorry sir this is left by mistake will remove it.